rpc/src/commands/mod.rs

```diff
@@ -1,10 +1,14 @@
+use crate::responses::Response;
 mod get_balance;
 mod get_blocks_in_chain;
 pub use get_balance::GetBalance;
 pub use get_blocks_in_chain::GetBlocksInChain;
 
 pub trait RPCClientCommand {
+    type R: Response;
```

Comment:
Maybe this should've been named something more descriptive. Thoughts?

Reply:
`R` seems pretty on par from the docs/examples of associated types. AFAIK single letter is encouraged unless it's mapped to a specific enum or variant (e.g. `type Err` for a `Result` variant).

On that topic, i'll also rename the modules to conform to the RFC 430 [naming conventions](https://rust-lang.github.io/api-guidelines/naming.html#naming) for acronyms:
> "In UpperCamelCase, acronyms and contractions of compound words count as one word: use Uuid rather than UUID, Usize rather than USize or Stdin rather than StdIn. In snake_case, acronyms and contractions are lower-cased: is_xid_start."

which means `RPC*` -> `Rpc*` across traits & structs.

Reply:
Sounds good. You're right that `R` is on par with associated type conventions, but I don't think its a good convention, since anyone reading code elsewhere will have to come here to know what its supposed to correspond to. Breaking convention might be worth it here for the sake of readability

Reply:
In my opinion it's more of an anti-pattern to stray away from the rust's "ubiquitous language" than to have a familiar albeit high-context name.

`R` for a `Response` feels as good as `T` does for a generic `Type`. I think this is totally fine. We can rely safely on the language's existing conventions, and by doing so we might make it even _easier_ for incoming contributors eventually.

Reply:
Having said this, if we're all on the same page its probably fine and might be better internally anyway.

Thoughts? @LazaroHurtado

Reply:
Originally I thought it should be left as is since it does follow convention and it would be pretty clear what 'R' represents after the first read through. Now I'm asking myself that if this repo was pulled and used for another project would the user be able to understand what 'R' is without having to look at the source code? Or if someone would want to add to our repo would they know what 'R' is after not touching the project for a while? I believe the answer is no, so with that I personally think we should break convention